packages/database/lib/migrations/20250310134848_connection_add_meta_fields.cjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    await knex.raw(`ALTER TABLE "_nango_connections"
+ADD COLUMN "credentials_expires_at" timestamptz,
+ADD COLUMN "last_refresh_success" timestamptz,
+ADD COLUMN "last_refresh_failure" timestamptz,
+ADD COLUMN "last_refresh_tries" int2,
+ADD COLUMN "refreshable" bool`);
+};
```

Comment:
do we need to add index(es)?

Reply:
I'm not sure yet which one will be best, I will add some after I change the code